code/modules/mechs/equipment/utility.dm

```diff
@@ -445,15 +445,11 @@
 
 	if (ismob(target))
 		var/mob/tmob = target
-		if (tmob.unacidable)
-			to_chat(user, SPAN_WARNING("\The [target] can't be drilled away."))
-			return
-		else
-			to_chat(tmob, FONT_HUGE(SPAN_DANGER("You're about to get drilled - dodge!")))
+		to_chat(tmob, FONT_HUGE(SPAN_DANGER("You're about to get drilled - dodge!")))
 
 	else if (isobj(target))
 		var/obj/tobj = target
-		if (tobj.unacidable)
+		if (!tobj.solvent_can_melt()) // why the hell does this use an acid check
```

Comment:
This should check `drill_head.material.hardness`, probably. (Later, the proc also uses `drill_head.material.brute_armor` which is an unpleasant abuse of variables...)